bodhi/tests/server/consumers/test_masher.py

```diff
@@ -25,6 +25,11 @@
 import tempfile
 import time
 import unittest
+try:
+    from http.client import IncompleteRead
+except ImportError:
+    # Python 2 does not have this Exception.
+    IncompleteRead = None
```

Comment:
You could import this directly in the test to save you from catching the ImportError

Reply:
That would violate PEP-8, so I prefer this style. Bodhi will [drop Python 2](https://github.com/fedora-infra/bodhi/issues/2759) soon anyway, so this code won't be here for long.